client/state/action-types.js

```diff
@@ -65,6 +65,11 @@ export const COUNTRY_STATES_REQUEST_SUCCESS = 'COUNTRY_STATES_REQUEST_SUCCESS';
 export const CURRENT_USER_FLAGS_RECEIVE = 'CURRENT_USER_FLAGS_RECEIVE';
 export const CURRENT_USER_ID_SET = 'CURRENT_USER_ID_SET';
 export const DESERIALIZE = 'DESERIALIZE';
+export const DIRECTLY_ASKING_QUESTION = 'DIRECTLY_ASK_QUESTION';
```

Comment:
The value should be the same as the variable here.

My vote is for `export const DIRECTLY_ASK_QUESTION = 'DIRECTLY_ASK_QUESTION';`